functions/src/modules/SQR.functions.ts

```diff
@@ -214,65 +215,71 @@ export const processSubmissions = functions.database.ref('/webforms/sqr/{submiss
 //          on FB (Storage Triggered)
 //
 /////////////////////////////////////////////////
+const path = require('path');
+const os = require('os');
+const fs = require('fs');
 
-export const updateEmailTemplates = functions.storage.object()
-.onFinalize(async object => {
-
-    const path = require('path');
-    const os = require('os');
-    const fs = require('fs');
-
-    const filePath = object.name;
-    // slice(0, -5) to get rid of the trailing '.html' extension
-    const fileName = filePath.split('/')[2].slice(0, -5);
-    const tempLocalFile = path.join(os.tmpdir(), fileName);
-    
+const updateEmailTemplates = async (filePath) => {
```

Comment:
You misunderstood the idea of moving all the email stuff into a separate file. This function and below — all should go to `email.functions.ts`. In `SQR` file you should only push to `/email/notifications`.